content/docs/start/metrics-parameters-plots.md

```diff
@@ -98,6 +98,34 @@ into `roc.json` for an additional plot.
 > structure of a metrics or plots file - it's pretty much user and case defined.
 > Please refer to `dvc metrics` and `dvc plots` for more details.
 
+You can view tracked metrics and plots with DVC. Let's start with the metrics:
+
+```dvc
+$ dvc metrics show
+Path         avg_prec    roc_auc
+scores.json  0.52048     0.9032
+```
+
+To view plots, first specify which arrays to use as the plot axes. We only need
+to do this once, and DVC will save our plot configurations.
+
+```dvc
+$ dvc plots modify prc.json -x recall -y precision
+Modifying stage 'evaluate' in 'dvc.yaml'
+$ dvc plots modify roc.json -x fpr -y tpr
+Modifying stage 'evaluate' in 'dvc.yaml'
+```
+
+Now let's view the plots:
+
+```dvc
+$ dvc plots show
+file:///Users/dvc/example-get-started/plots.html
+```
+
+![](/img/plots_prc_get_started_show.svg)
```

Comment:
can we try to do them side by side? they take quite a lot of space at the moment

Reply:
Agreed. Any advice on how to do that? Putting the two on the same line has no impact, and I'm not sure how to resize given that they are SVGs. I'm assuming it's controlled by the web framework, and I don't know why these are on separate lines when the `diff` plots below fit on one. It looks like the `show` files are bigger than the `diff` SVGs for some reason.

Reply:
> Any advice on how to do that?

You could hardcode some of the HTML in https://dvc.org/doc that creates columns (on certain screen widths).

Now and then we stumble on the need to make 2-col sections, maybe we can think about introducing an engine customization for this @rogermparent ? Like our custom `<details/abbr>` tags for example (just not sure which tag to customize.

Reply:
Wait... Dah there's an easier way: Just combine both plots into one in an img editing app 🙂 I think you're already doing that in #2195?

Reply:
No, the other images just happen to fit for some reason despite also being two separate images. How would I combine two SVG files? Is another acceptable format I can convert to?

Reply:
okay, managed to get them smaller (just edited SVGs directly)